css/style.css

```diff
@@ -13,6 +13,10 @@ font-family: 'IBM Plex Sans', sans-serif;
 
 /* Resets */
 
+html {
+  scroll-behavior: smooth;
```

Comment:
Nice, if you want to see more details and how this property affects your page, please take a look at this: https://css-tricks.com/almanac/properties/s/scroll-behavior/

Reply:
Reviewed. Thanks.